### PR TITLE
[RESTinio] Update to v.0.6.8.1

### PR DESCRIPTION
**Pull request by eao197**
RESTinio updated to the latest version with several fixes.

### CODE REVIEW

File: ports/restinio/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO stiffstream/restinio
-    REF c34f83a68e60537eba52e2d076ed22495ad4c5df # v.0.6.8
-    SHA512 1c03a2cc96f44ce004870136ffdaa7a9abc5bc5173edadfe58e4f92f2e3c67c1555b4604094ad3a1dab1f0bf01cf9d79cf7d5a381f357aa4bdff90656a27e0c1
+    REF c4ed2d0956948d47a8fd569f3c0f3d2113219421 # v.0.6.8
```
Comment (c72578) [inline ports/restinio/portfile.cmake:4]: # v.0.6.8 -> # v.0.6.8.1

Comment (eao197) [reply ports/restinio/portfile.cmake]: Thanks! Fixed in 4a8dea9

**Review by PhoebeHui — APPROVED** [review]
Thanks for the PR!

**Comment by strega-nil** [comment]
Oh, oops, I was looking at the wrong tests -- this passed all modern tests.

**Pull request merged**